js/login.js

```diff
@@ -0,0 +1,41 @@
+/* variables */
+
+let logIn = document.getElementById("login-btn");
+let loginForm = document.querySelector(".login-form");
+
+logIn.addEventListener("click", (e) => {
+    e.preventDefault();
+
+
+    let emailId = document.getElementById('email').value;
```

Comment:
```suggestion
    const emailId = document.getElementById('email').value;
```
Instead of using let for variables that won't be reassigned, use const